Index: app/src/main/java/com/application/anicaremals/ui/premium/roughidea.kt
===================================================================
diff --git a/app/src/main/java/com/application/anicaremals/ui/premium/roughidea.kt b/app/src/main/java/com/application/anicaremals/ui/premium/roughidea.kt
deleted file mode 100644
--- a/app/src/main/java/com/application/anicaremals/ui/premium/roughidea.kt	(revision 8ac475c1fc00b2bdc98d1df0444eddb4cc853b02)
+++ /dev/null	(revision 8ac475c1fc00b2bdc98d1df0444eddb4cc853b02)
@@ -1,4 +0,0 @@
-package com.application.anicaremals.ui.premium
-
-class roughidea {
-}
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"DesignSurface\">\r\n    <option name=\"filePathToZoomLevelMap\">\r\n      <map>\r\n        <entry key=\"..\\:/Android Development/Projects/AniCareMals/version_0.1/app/src/main/res/layout/activity_main.xml\" value=\"0.38697916666666665\" />\r\n      </map>\r\n    </option>\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_11\" default=\"true\" project-jdk-name=\"Android Studio default JDK\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/build/classes\" />\r\n  </component>\r\n  <component name=\"ProjectType\">\r\n    <option name=\"id\" value=\"Android\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 8ac475c1fc00b2bdc98d1df0444eddb4cc853b02)
+++ b/.idea/misc.xml	(date 1637323077781)
@@ -7,7 +7,7 @@
       </map>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="Android Studio default JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="11" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
